libs/single-spa-angular/parcel/src/parcel.component.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  Component,
+  ElementRef,
+  Input,
+  OnChanges,
+  OnDestroy,
+  OnInit,
+  ChangeDetectionStrategy,
+} from '@angular/core';
+import { Parcel, ParcelConfig, AppProps } from 'single-spa';
+
+const enum Action {
+  Mount = 'mount',
+  Update = 'update',
+  Unmount = 'unmount',
+}
+
+@Component({
+  selector: 'parcel',
+  template: '',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ParcelComponent implements OnChanges, OnInit, OnDestroy {
+  @Input() config: ParcelConfig | null = null;
+  @Input() mountParcel: AppProps['mountParcel'] | null = null;
+  @Input() onParcelMount: (() => void) | null = null;
+  @Input() wrapWith = 'div';
+  @Input() customProps: object = {};
+  @Input() appendTo: Node | null = null;
+  @Input() handleError = (error: Error) => console.error(error);
+
+  private hasError = false;
+  private unmounted?: boolean;
+  private wrapper: HTMLElement | null = null;
+  private parcel: Parcel | null = null;
+  private task: Promise<void> | null = null;
+
+  constructor(private host: ElementRef<HTMLElement>) {}
+
+  ngOnChanges(): void {
+    this.scheduleTask(Action.Update, () => {
+      if (this.parcel !== null && this.parcel.update) {
+        return this.parcel.update(this.customProps);
+      }
+    });
+  }
+
+  ngOnInit(): void {
+    this.scheduleTask(Action.Mount, () => {
+      if (this.mountParcel === null) {
+        throw new Error(`
+				  <parcel> was not passed a [mountParcel] binding.
+				  If you are using <parcel> within a module that is not a single-spa application, you will need to import mountRootParcel from single-spa and pass it into <parcel> as a [mountParcel] binding
```

Comment:
One thing to note here, is that there is `mountRootParcel` and `mountParcel`. The `mountParcel` function is one that is provided as a prop to all single-spa applications. The two functions are identical, except that `mountParcel` is bound to the current application, and will automatically unmount the parcel when the application is unmounted.

In React, we automatically use the `mountParcel` function inside of single-spa-react by using React context to do dependency injection.

Let's update the languaging here to reflect this:

```suggestion
				  If you are using <parcel> within a module that is not a single-spa application, you will need to either (1) import mountRootParcel from single-spa or (2) use the `mountParcel` prop provided to single-spa applications. One of those two should be passed into <parcel> as a [mountParcel] binding
```